components/ledger/cmd/script_test.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/formancehq/stack/libs/go-libs/api"
+	"github.com/google/uuid"
+	"github.com/numary/ledger/pkg/api/controllers"
+	"github.com/numary/ledger/pkg/core"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ScriptCommands(t *testing.T) {
```

Comment:
From a methodology point of view, these kind of tests are well suited for table driven tests (https://dave.cheney.net/2019/05/07/prefer-table-driven-tests)